packages/aws-rfdk/README.md

```diff
@@ -15,4 +15,4 @@ Please see the following sources for additional information:
 * The [RFDK API Documentation](https://docs.aws.amazon.com/rfdk/api/latest/docs/aws-rfdk-construct-library.html)
 * The [README for the main module](https://github.com/aws/aws-rfdk/blob/mainline/packages/aws-rfdk/lib/core/README.md)
 * The [README for the Deadline module](https://github.com/aws/aws-rfdk/blob/mainline/packages/aws-rfdk/lib/deadline/README.md)
-* The [RFDK Upgrade Documentation](./docs/upgrade/index.md)
+* The [RFDK Upgrade Documentation](https://github.com/aws/aws-rfdk/blob/mainline/packages/aws-rfdk/docs/upgrade/index.md)
```

Comment:
Maybe a bit of a nitpick, but it would be better, imo, if the links were to the release branch instead of mainline (the active dev branch).

Reply:
Yeah, that's a good idea. Updated all our links to point to the release branch